packages/firestore/src/core/firestore_client.ts

```diff
@@ -202,7 +202,8 @@ export class FirestoreClient {
   enableNetwork(): Promise<void> {
     this.verifyNotTerminated();
     return this.asyncQueue.enqueue(() => {
-      return this.syncEngine.enableNetwork();
```

Comment:
I removed the FirestoreClient -> SyncEngine -> LocalStore -> Persistence indirection for `enableNetwork` and `disableNetwork` since I didn't want to come up with non-conflicting global names for the new free-standing functions.